operations/src/test/java/com/nextdoor/bender/operation/delete/DeleteFieldOperationTest.java

```diff
@@ -0,0 +1,41 @@
+package com.nextdoor.bender.operation.delete;
+
+import com.google.gson.JsonObject;
+import com.nextdoor.bender.InternalEvent;
+import com.nextdoor.bender.deserializer.json.GenericJsonEvent;
+import org.junit.Test;
+
+public class DeleteFieldOperationTest {
+
+    @Test(expected = NullPointerException.class)
+    public void testNullField() {
+        InternalEvent ievent = new InternalEvent("foo", null, 1);
+
+        DeleteFieldOperation operation = new DeleteFieldOperation(null);
+        operation.perform(ievent);
+    }
+
+    @Test
+    public void testExistingField() {
+        InternalEvent ievent = new InternalEvent("foo", null, 1);
+        JsonObject jsonObject = new JsonObject();
+        jsonObject.addProperty("ex", "exists");
+        GenericJsonEvent jsonEvent = new GenericJsonEvent(jsonObject);
+        ievent.setEventObj(jsonEvent);
+
+        DeleteFieldOperation operation = new DeleteFieldOperation("ex");
+        operation.perform(ievent);
```

Comment:
On all of these tests you'll want to add an assert to verify the property was actually deleted. See this as example of how to do that https://github.com/Nextdoor/bender/blob/master/operations/src/test/java/com/nextdoor/bender/operation/json/array/ArraySplitOperationTest.java#L51

Reply:
Thanks, adding this.